content/projects/dags-with-airflow/_index.md

```diff
@@ -11,6 +11,10 @@ ready: true
 submission_type: repo
 tags:
 - apache-airflow
+- postgresql
+- sql
+- sqlite
+- .gitignore
```

Comment:
1. why is the .gitignore tag relevant here?
2. Should this not be tagged `dags`?

Reply:
not sure if i am right here so if I am wrong just reply on the comment :)